src/JustSaying/AwsTools/MessageHandling/Dispatch/MessageDispatcher.cs

```diff
@@ -29,13 +30,15 @@ internal class MessageDispatcher : IMessageDispatcher
         public MessageDispatcher(
             IMessageSerializationRegister serializationRegister,
             IMessageMonitor messagingMonitor,
+            Action<Exception, SQSMessage> onError,
```

Comment:
Disadvantage of this it that it doesn't fit well with DI.

Reply:
You might be looking at the wrong PR here, I accidentally pushed it here but its separate for now to get thoughts: https://github.com/gkinsman/JustSaying/pull/32